lib/debootstrap.sh

```diff
@@ -628,8 +629,15 @@ PREPARE_IMAGE_SIZE
 		parted -s ${SDCARD}.raw -- mkpart primary ${parttype[$ROOTFS_TYPE]} ${rootstart}s "100%"
 	else
 		# /boot partition + root partition
-		parted -s ${SDCARD}.raw -- mkpart primary ${parttype[$bootfs]} ${bootstart}s ${bootend}s
-		parted -s ${SDCARD}.raw -- mkpart primary ${parttype[$ROOTFS_TYPE]} ${rootstart}s "100%"
+		local parm1='primary'
+		local parm2=${parm1}
```

Comment:
Please use meaningful names.

Reply:
corrected. It did however call my attention to another variable array in this script, called parttype. Actually that should be called file system type, so fstype. But was asked to touch the rest as little as possible as it may have unwanted effects elsewhere.

Reply:
Yes. It maybe a historical issues. Due to there are two hook `pre_prepare_partitions` and `prepare_partitions_custom` which allow custom options. So it maybe cause some problems